controlplane/kubeadm/controllers/helpers.go

```diff
@@ -92,6 +101,24 @@ func (r *KubeadmControlPlaneReconciler) reconcileKubeconfig(ctx context.Context,
 	return nil
 }
 
+func (r *KubeadmControlPlaneReconciler) adoptKubeconfigSecret(ctx context.Context, cluster *clusterv1.Cluster, configSecret *corev1.Secret, controllerOwnerRef metav1.OwnerReference) error {
```

Comment:
Should we add a log line indicating an adoption of a secret?

Reply:
done!